migrations/20200516011910-initial-data-load.js

```diff
@@ -0,0 +1,312 @@
+
+module.exports = {
+  up: async (queryInterface) => {
+    /*
+      Add altering commands here.
+      Return a promise to correctly handle asynchronicity.
+
+      Example:
+      return queryInterface.createTable('users', { id: Sequelize.INTEGER });
+    */
+    await queryInterface.bulkInsert('authors', [
+      {
+        nameFirst: 'Bram',
+        nameLast: 'Stoker'
+      }, {
+        nameFirst: 'Oscar',
+        nameLast: 'Wilde'
+      }, {
```

Comment:
I would one line these to reduce the amount of scrolling needed. As long as you are consistent ESLint will allow it.

```JavaScript
{ nameFirst: 'Bram', nameLast: 'Stoker' },
{ nameFirst: 'Oscar', nameLast: 'Wilde' },
```